git_sim/__main__.py

```diff
@@ -76,9 +77,18 @@ def main():
 
     args = parser.parse_args()
 
+    if sys.platform == 'linux' or sys.platform == 'darwin':
+        repo_name = git.Repo(search_parent_directories=True).working_tree_dir.split('/')[-1]
+    elif sys.platform == 'win32':
+        repo_name = git.Repo(search_parent_directories=True).working_tree_dir.split('\\')[-1]
+
     config.media_dir = os.path.join(os.path.expanduser(args.media_dir), "git-sim_media")
     config.verbosity = "ERROR"
 
+    # If the env variable is set and no argument provideed, use the env variable value
+    if os.getenv('git_sim_media_dir') is not None and args.media_dir == '.':
```

Comment:
I think we can shorten the `if` statement if we remove the `is not None` and just use `if os.getenv('git_sim_media_dir') and args.media_dir == '.':`